config/releases.exs

```diff
@@ -64,6 +64,12 @@ config :sentry,
 config :ueberauth, Ueberauth.Strategy.Cognito, client_secret: cognito_client_secret
 config :screens, ScreensWeb.AuthManager, secret_key: screens_auth_secret
 
+config :screens, ScreensByAlert.Memcache,
+  connection_opts: [
+    namespace: System.get_env("HOST"),
+    hostname: System.get_env("MEMCACHED_HOST")
+  ]
+
```

Comment:
Will this work with the `@cache_module` module attribute? I thought `releases.exs` is a runtime config file but I believe module attributes are evaluated/expanded at compile time 🤔

Reply:
`@cache_module` doesn't need this particular piece of the config. It only needs what is in `prod.exs` (which I just realized was missing 😬). `connection_opts` is used when the Memcached instance is booting up in the application's supervision tree, on [this line](https://github.com/mbta/screens/blob/298213606796724633bc393c5ff5beb173d6e852/lib/screens/screens_by_alert/memcache.ex#L12) specifically.

Reply:
Ah yeah woops, I must've misread something. Makes sense!